plugin.video.themoviedb.helper/.gitignore

```diff
@@ -0,0 +1,63 @@
+# Created by .gitignore support plugin (hsz.mobi)
```

Comment:
please remove from submission

Reply:
Whoops! Thought I already did. Must've only deleted from the Leia version.

Done!